doc/md/privacy.md

```diff
@@ -545,22 +548,22 @@ func (Group) Policy() ent.Policy {
 
 Then, we expect the privacy-rules to take effect on the client operations.
 
-```go
+```go title="examples/privacytenant/example_test.go"
 func Do(ctx context.Context, client *ent.Client) error {
     // A continuation of the code-block above.
 
-    // Expect operation to fail, because the FilterTenantRule rule makes sure
-    // that tenants can update and delete only their groups.
-    err = entgo.Update().SetName("fail.go").Exec(labView)
-    if !ent.IsNotFound(err) {
-    	return fmt.Errorf("expect operation to fail, since the group (entgo) is managed by a different tenant (hub)")
-    }
-    entgo, err = entgo.Update().SetName("entgo").Save(hubView)
-    if err != nil {
-    	return fmt.Errorf("expect operation to pass, but got %w", err)
-    }
-    fmt.Println(entgo)
-    return nil
+	// Expect operation to fail, because the FilterTenantRule rule makes sure
+	// that tenants can update and delete only their groups.
+	if err = entgo.Update().SetName("fail.go").Exec(labView); !ent.IsNotFound(err) {
+		return fmt.Errorf("expect operation to fail, since the group (entgo) is managed by a different tenant (hub), but got %w", err)
+	}
+	entgo, err = entgo.Update().SetName("entgo").Save(hubView)
+	if err != nil {
+		return fmt.Errorf("expect operation to pass, but got %w", err)
+	}
+	fmt.Println(entgo)
+
+	return nil
```

Comment:
```suggestion
	err = entgo.Update().SetName("fail.go").Exec(labView)
	// Expect operation to fail, because the FilterTenantRule rule makes
	// sure that tenants can update and delete only their groups.
	if !ent.IsNotFound(err) {
		return fmt.Errorf("expect operation to fail, since the group (entgo) is managed by a different tenant (hub), but got %w", err)
	}
	entgo, err = entgo.Update().SetName("entgo").Save(hubView)
	if err != nil {
		return fmt.Errorf("expect operation to pass, but got %w", err)
	}
	fmt.Println(entgo)
	return nil
```